src/MFRC522Extended.cpp

```diff
@@ -828,19 +824,19 @@ MFRC522::StatusCode MFRC522Extended::TCL_Transceive(TagInfo *tag, byte *sendData
 		*backLen = in.inf.size;
 		memcpy(backData, in.inf.data, in.inf.size);
 	}
-
+	
 	// Check chaining
 	if ((in.prologue.pcb & 0x10) == 0x00)
 		return result;
-
 	// Result is chained
 	// Send an ACK to receive more data
 	// TODO: Should be checked I've never needed to send an ACK
+
 	while (in.prologue.pcb & 0x10) {
```

Comment:
actually, the variables "in"s are local variables. So `while (in.prologue.pcb & 0x10)` in function `TCL_Transceive` will always be true although `in.prologue.pcb` in function `TCL_TransceiveRBlock ` will change.